mteb/tasks/Retrieval/eng/LEMBNarrativeQARetrieval.py

```diff
@@ -0,0 +1,67 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBNarrativeQARetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "test"
+
+    metadata = TaskMetadata(
+        name="LEMBNarrativeQARetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "narrativeqa",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("narrativeqa subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=["written"],
+        domains=["Fiction", "Non-fiction"],
+        task_subtypes=["Article retrieval"],
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators="derived",
+        dialect=None,
```

Comment:
assuming there are no dialects:
```suggestion
        dialect=[],
```

Reply:
great

Reply:
Thanks a lot for you elaboration! @KennethEnevoldsen